source/app/business/case_comments.py

```
@@ -32,8 +31,8 @@ def case_comments_update(comment_text, comment_id, object_type, caseid):
     if not comment:
         raise BusinessProcessingError('Invalid comment ID')
 
-    if hasattr(current_user, 'id') and current_user.id is not None:
-        if comment.comment_user_id != current_user.id:
+    if hasattr(iris_current_user, 'id') and iris_current_user.id is not None:
+        if comment.comment_user_id != iris_current_user.id:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)